hub/static/js/explore.esm.js

```diff
@@ -210,11 +207,41 @@ const app = createApp({
           }).addTo(this.map)
         })
     },
+    updateMap() {
+      if (!this.map) { this.setUpMap().then( this.filterMap ) } else { this.filterMap() }
+    },
+    filterMap() {
+      fetch(this.url('/explore.json'))
```

Comment:
@struan FYI I getting a Django exception when I filter by Urban Rural Classification:

<img width="476" alt="Screenshot 2022-12-14 at 14 28 31" src="https://user-images.githubusercontent.com/739624/207622157-98c0c7f7-922d-4182-927a-ae283330e20a.png">

```
FieldError at /explore.json

Cannot resolve keyword 'None' into field. Choices are: area_type, areadata, geometry, gss, id, mapit_id, name, person

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/views/generic/base.py", line 226, in get
    return self.render_to_response(context)
  File "/app/hub/views.py", line 125, in render_to_response
    areas = list(self.query().filter(geometry__isnull=False))
  File "/app/hub/mixins.py", line 48, in query
    query = f["dataset"].filter(
  File "/app/hub/models.py", line 171, in filter
    return Filter(self, query).run(**kwargs)
  File "/app/hub/filters.py", line 24, in run
    return self.query.exclude(**kwargs) if exclude else self.query.filter(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/query.py", line 1420, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/query.py", line 1438, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/query.py", line 1445, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1532, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1562, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/usr/local/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1407, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1217, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/usr/local/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1709, in names_to_path
    raise FieldError(
```

Reply:
what does the admin page for this look like?

Reply:
![Screenshot 2022-12-15 at 08-56-51 Urban Rural Classification Change data set Django site admin](https://user-images.githubusercontent.com/739624/207816204-8d7b8632-32b7-4f3b-af24-279963fd1b2e.png)

Reply:
If you update the table field in there to areadata then it should work